cmd/helm/dependency_build_test.go

```diff
@@ -100,3 +100,22 @@ func TestDependencyBuildCmd(t *testing.T) {
 		t.Errorf("mismatched versions. Expected %q, got %q", "0.1.0", v)
 	}
 }
+
+func TestDependencyBuildCmdWithHelmV2Hash(t *testing.T) {
+	chartName := "testdata/testcharts/issue-7233"
+
+	cmd := fmt.Sprintf("dependency build '%s'", chartName)
+	_, out, err := executeActionCommand(cmd)
+
+	// Want to make sure the build can verify Helm v2 hash
+	if err != nil {
+		t.Logf("Output: %s", out)
+		t.Fatal(err)
+	}
+
+	// Make sure the child chart file got downloaded.
+	expect := filepath.Join(chartName, "charts/alpine-0.1.0.tgz")
```

Comment:
The chart's committed. So if the download doesn't happen, this still passes? Or does this logic rely on `helm dep build` removing the existing file first?

Or is alpine-0.1.0.tgz supposed to be committed outside this Chart, so that `helm dep build` copies it in?

Reply:
Hmm, good point. Removing the check as it doesn't make lot of sense as including the tarball.